src/index.html

```diff
@@ -7,6 +7,466 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="./images/logo.png" alt="moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+                Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main_container">
+        <div
+        class="card"
+        data-qa="card"
+        >
```

Comment:
```suggestion
        <div
          class="card"
          data-qa="card"
        >
```
Please, fix the code style everywhere.

Reply:
done